### DISCUSSION TITLE
TypeError: Object(...) is not a function

### DISCUSSION THREAD

**Discussion by sankparatkar** [react-hook-form/react-hook-form#7727] [Q&A]
I am getting this error when I am trying to use useForm() with latest version of react-hook-form
My react version is 16.8.3, but the same code works in another project with same react version so that does not seem to be an issue.
I can see redux and redux persist persist in stack trace, might have some clashes while mounting not so sure.
Please advise

Code Snippet :

`import React from "react";
import { useForm } from "react-hook-form";

export default function Test() {
const { register, handleSubmit, formState: { errors } } = useForm();

return (
{/* JSX from the sample code https://react-hook-form.com/get-started */}

</form>
);
}`

Stack Trace :

renderWithHooks
../node_modules/react-dom/cjs/react-dom.development.js:14797
mountIndeterminateComponent
../node_modules/react-dom/cjs/react-dom.development.js:17409
beginWork
../node_modules/react-dom/cjs/react-dom.development.js:18489
HTMLUnknownElement.callCallback
../node_modules/react-dom/cjs/react-dom.development.js:189
invokeGuardedCallbackDev
../node_modules/react-dom/cjs/react-dom.development.js:238
invokeGuardedCallback
../node_modules/react-dom/cjs/react-dom.development.js:291
beginWork$1
../node_modules/react-dom/cjs/react-dom.development.js:23055
performUnitOfWork
../node_modules/react-dom/cjs/react-dom.development.js:22019
workLoopSync
../node_modules/react-dom/cjs/react-dom.development.js:21995
performSyncWorkOnRoot
../node_modules/react-dom/cjs/react-dom.development.js:21613
(anonymous function)
../node_modules/react-dom/cjs/react-dom.development.js:11131
unstable_runWithPriority
../node_modules/scheduler/cjs/scheduler.development.js:659
runWithPriority$1
../node_modules/react-dom/cjs/react-dom.development.js:11077
flushSyncCallbackQueueImpl
../node_modules/react-dom/cjs/react-dom.development.js:11126
flushSyncCallbackQueue
../node_modules/react-dom/cjs/react-dom.development.js:11114
scheduleUpdateOnFiber
../node_modules/react-dom/cjs/react-dom.development.js:21056
enqueueSetState
../node_modules/react-dom/cjs/react-dom.development.js:12676
PersistGate.push../node_modules/react/cjs/react.development.js.Component.setState
../node_modules/react/cjs/react.development.js:464
(anonymous function)
../node_modules/redux-persist/es/integration/react.js:137
dispatch
../node_modules/redux/es/redux.js:224
rehydrate
../node_modules/redux-persist/es/persistStore.js:136
_rehydrate
../node_modules/redux-persist/es/persistReducer.js:136
(anonymous function)
../node_modules/redux-persist/es/persistReducer.js:165

**Comment by Moshyfawn** [comment]
I don't think we can magically see what's the issue with your environment.
I've created [this CSB](https://codesandbox.io/s/redux-persist-example-forked-bgyjd?file=/index.js) where RHF is being used along side the `redux` and `redux-persist`. You can try tweaking it to match your use-case better to hopefully reproduce the issue
